backend/de.metas.qualitymgmt/src/main/java/de/metas/qm/analysis/QMAnalysisReportDocumentHandlerProvider.java

```diff
@@ -28,19 +28,22 @@
 import org.springframework.stereotype.Component;
 
 @Component
-public class QMAnalysisReportDocumentHandlerProvider  implements DocumentHandlerProvider
-{
+public class QMAnalysisReportDocumentHandlerProvider implements DocumentHandlerProvider {
```

Comment:
1. pls use our code formatter.
2. annotate the class with `@RequiredArgsConstructor` and get rid of the constructor

Reply:
done